src/solver/conv_winoRxS.cpp

```diff
@@ -190,31 +189,41 @@ inline bool IsShaderContraintsMet(const ExecutionContext& ctx,
         if(!(0 <= problem.GetBackwardPadH() && problem.GetBackwardPadH() < std::pow(2, 16)))
             return false;
     }
-    const auto grid_workgroup_count_x = ctx.GetStream().GetMaxHardwareComputeUnits();
     if(!problem.IsLayoutDefault())
     {
         return false;
     }
 
+    uint64_t o_K_stride      = OH * OW;
```

Comment:
@johnny-keker  @atamazov interestingly this line was not caught by CI, but failed CI after being merged into develop branch :(

Reply:
@junliume I recommend issuing a team-wide reminder to update their branches from develop after updating Jenkinsfile, Dockerfile or tests.